gradle/maven.gradle

```diff
@@ -1,4 +1,4 @@
-apply plugin: 'maven'
+//apply plugin: 'maven'
```

Comment:
- [ ] Update the release process in this PR or create a new ticket to do that later.